packages/swingset-liveslots/src/virtualObjectManager.js

```diff
@@ -954,6 +954,14 @@ export const makeVirtualObjectManager = (
         makeContextProviderKit(contextCache, getSlotForVal, facetNames),
         behavior,
         thisfulMethods,
+        // If annotating with the following at-ts-expect-error, then
+        // linting packages/telemetry complains of an unused
+        // at-ts-expect-error. But if it is removed, then linting
+        // here (swingset-liveslots) complains.
+        // at-ts-expect-error TS does not know it can only be a kit or undefined
+        //
+        // eslint-disable-next-line @typescript-eslint/prefer-ts-expect-error
+        // @ts-ignore
```

Comment:
I bet we can get these to agree but until then:
```suggestion
    // eslint-disable-next-line @typescript-eslint/prefer-ts-expect-error -- https://github.com/Agoric/agoric-sdk/issues/4620
    // @ts-ignore TS does not know it can only be a kit or undefined
```

Reply:
With your previous suggestion, these type errors disappeared. Good riddance!